### PR TITLE
behnaz/Dmt5-isready2

### PR DESCRIPTION
**Pull request by behnaz-fs**

### CODE REVIEW

File: html/broadcasts/rebranding/DMT5-isready.html [hunk 1]
```diff
@@ -165,39 +165,39 @@
...
+                                <p style="font-family: 'IBM Plex Sans', Arial, sans-serif; color: #333333; font-size: 16px; font-weight: 400; line-height: 24px; margin: 0px 0px 0px 0px;">Need help? Please <span style="color: #ff444f;"><a href="mailto:support@deriv.com">let us know.</a></span></p>
                                 <p style="font-family: 'IBM Plex Sans', Arial, sans-serif; color: #333333; font-size: 16px; font-weight: 400; line-height: 24px; margin: 8px 0px 0px 0px;">Thank you for choosing us!</p>
                             </td>
                         </tr>
                         <tr>
                             <td bgcolor="#ffffff" align="center" style="padding: 40px 30px 40px 30px; background-color:#ffffff;">
-                                <img src="https://static.deriv.com/html/broadcasts/rebranding/images/salute.png" width="240" height="24" border="0" style="display: block;" alt="">
+                                <img src="https://static.deriv.app/html/images/social/dark/salute.png" width="32px" height="32px" border="0" style="display: block;" alt="Salute hand-logo">
```
Comment (sash-binary) [inline html/broadcasts/rebranding/DMT5-isready.html:200]: https://static.deriv.com

File: html/broadcasts/rebranding/DMT5-isready.html [hunk 2]
```diff
@@ -165,39 +165,39 @@
...
                                 <p style="font-family: 'IBM Plex Sans', Arial, sans-serif; color: #333333; font-size: 16px; font-weight: 400; line-height: 24px; margin: 10px 0px 0px 0px;">If you wish to create another DMT5 account, you can simply do it from your DMT5 dashboard.</p>
                             </td>
                         </tr>
                         <!-- button -->
                         <tr>
-                            <td bgcolor="#ffffff" align="center" style="padding: 0px 30px 50px 30px; color: #333333; font-family: 'IBM Plex Sans', Arial, sans-serif; font-size: 18px; font-weight: 400; line-height: 24px;">
+                            <td bgcolor="#ffffff" align="center" style="padding: 0px 30px 30px 30px; color: #333333; font-family: 'IBM Plex Sans', Arial, sans-serif; font-size: 18px; font-weight: 400; line-height: 24px;">
                                 <!--[if (gte mso 9)|(IE)]>
                                     <v:roundrect xmlns:v="urn:schemas-microsoft-com:vml" xmlns:w="urn:schemas-microsoft-com:office:word" href="https://www.deriv.com" style="height:40px;v-text-anchor:middle;width:180px;" arcsize="8%" stroke="f" fillcolor="#ff444f">
```
Comment (sash-binary) [inline html/broadcasts/rebranding/DMT5-isready.html:184]: update outlook link url, width, style and name please

File: html/broadcasts/rebranding/DMT5-isready.html (hunk 1 above)
Comment (sash-binary) [reply html/broadcasts/rebranding/DMT5-isready.html:200]: change all instances of .app to .com https://static.deriv.app

File: html/broadcasts/rebranding/DMT5-isready.html (hunk 2 above)
Comment (sash-binary) [reply html/broadcasts/rebranding/DMT5-isready.html:184]: footer copyrights top spacing and font color are not correct, please update

File: html/broadcasts/rebranding/DMT5-isready.html (hunk 1 above)
Comment (behnaz-fs) [reply html/broadcasts/rebranding/DMT5-isready.html:198]: done

File: html/broadcasts/rebranding/DMT5-isready.html (hunk 2 above)
Comment (behnaz-fs) [reply html/broadcasts/rebranding/DMT5-isready.html:188]: done

**Pull request merged**